data/portal/Resource counts.csv

```diff
@@ -0,0 +1,11 @@
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
+Resource counts,,,,,,,,,,,,Quantitative information on the resource,"DataCatalogueFlat,RWEStaging,StudiesStaging",
+Resource counts,,resource,ref,1,,,Resources,,,,,Resource id,"DataCatalogueFlat,RWEStaging,StudiesStaging",
+Resource counts,,age group,ontology,1,,CatalogueOntologies,Age groups,,,,,Select the relevant age group for this quantitative information,"DataCatalogueFlat,RWEStaging,StudiesStaging",
+Resource counts,,population size,int,,,,,,,,,"Total number of unique individuals with records captured in the data source (most recent count). In the catalogue, this will accommodate counts per year","DataCatalogueFlat,RWEStaging,StudiesStaging",
+Resource counts,,active size,int,,,,,,,,,"Number of unique, active, or currently registered individuals with records captured in the data source (most recent count). In the catalogue, this will accommodate counts per year","DataCatalogueFlat,RWEStaging,StudiesStaging",
+Resource counts,,no individuals with samples,int,,,,,,,,,"Number of unique individuals with records of biological samples (e.g., blood, urine) (most recent count). In the catalogue, this will accommodate counts per year ","DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging",
```

Comment:
If this table refers to groups of individuals, could this be shortened `number of samples`? At the very least, this should be `num of individuals with samples` to prevent any confusion as `no` reads as *number of individuals with no samples*

Reply:
add 'number of unique samples'. And rename to 'number of individuals with samples'\

add to design document that we don't use abbreviations (guiding in principle) to prevent confusion.